framework/utils/mathUtils.py

```diff
@@ -1110,3 +1110,30 @@ def computeCrowdingDistance(trainSet):
 
   crowdingDist = np.sum(distMat,axis=1)
   return crowdingDist
+
+
+def getBuiltinTypes(typ):
+  """
+    Method to get a dictionary of builtin types
+    @ In, typ, str, the type to get
+    @ Out, bTypes, list, the list of type instances
+  """
+  import builtins as b
+  bTypes = {k:[t] for k, t in b.__dict__.items() if isinstance(t, type)}.get(typ,[])
```

Comment:
could we make this dict once then reference it in this method, rather than rebuild the dict each call?

similar for the numpy types

Reply:
done